src/lib/urlUtils.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/prefer-default-export */
+// regex to match a valid url, it must start with http:// or https://, have at least one dot, and not end with a dot or a space
+const VALID_URL_REGEX = /^https?:\/\/.*\.[^ |^.]/;
```

Comment:
This is the regex i needed to cleanly pull urls out of the legacy data.
```suggestion
const VALID_URL_REGEX = '(?:(?:http|ftp|https|file):\\/\\/)?(?:[\\w%_-]+(?:(?:\\.[\\w%_-]+)+)|(?:\\/[\\w][:]))(?:[\\w\\\\\'\'.,@?^=%&:\\/~+#()-]*[\\w@?^=%&\\/~+#-])';
```